docs/reference/settings/ml-settings.asciidoc

```diff
@@ -134,19 +136,19 @@ enabled, this setting can be updated only by operator users.
 
 `xpack.ml.max_anomaly_records`::
 (<<cluster-update-settings,Dynamic>>) The maximum number of records that are
-output per bucket. The default value is `500`.
+output per bucket. Defaults to `500`.
 
 `xpack.ml.max_lazy_ml_nodes`::
 (<<cluster-update-settings,Dynamic>>) The number of lazily spun up {ml} nodes.
 Useful in situations where {ml} nodes are not desired until the first {ml} job
-opens. It defaults to `0` and has a maximum acceptable value of `3`. If the
-current number of {ml} nodes is greater than or equal to this setting, it is
-assumed that there are no more lazy nodes available as the desired number
-of nodes have already been provisioned. If a job is opened and this setting has
-a value greater than zero and there are no nodes that can accept the job, the
-job stays in the `OPENING` state until a new {ml} node is added to the cluster
-and the job is assigned to run on that node. When the {operator-feature} is
-enabled, this setting can be updated only by operator users.
+opens. If the current number of {ml} nodes is greater than or equal to this 
+setting, it is assumed that there are no more lazy nodes available as the
+desired number of nodes have already been provisioned. If a job is opened and 
+this setting has a value greater than zero and there are no nodes that can
+accept the job, the job stays in the `OPENING` state until a new {ml} node is 
+added to the cluster and the job is assigned to run on that node. When the
+{operator-feature} is enabled, this setting can be updated only by operator
+users. The maximum value is `3`. Defaults to `0`.
```

Comment:
There is no longer a maximum of 3 in 7.11 and above.  This was changed to support autoscaling.

```suggestion
users. Defaults to `0`.
```